examples/petstore/lib/server_test.go

```diff
@@ -29,5 +29,5 @@ func TestPetstoreOpenAPIGeneration(t *testing.T) {
 	generatedSpec, err := os.ReadFile("testdata/doc/openapi.json")
 	require.NoError(t, err)
 
-	golden.Assert(t, string(generatedSpec), "doc/openapi.golden.json")
+	golden.Assert(t, string(generatedSpec), "doc/openapi.json")
```

Comment:
No, please compare against the golden file.

If there are errors, please run `make golden-update`. It will show you the effect of your PR on an existing codebase